app/[user]/page.tsx

```diff
@@ -3,7 +3,9 @@ import { headers } from "next/headers";
 import Image from 'next/image';
 import Navbar from "../components/Navbar";
 import { EditProfileAndFollowButton } from "../components/ClientUtilityFunctions";
-// TODO These imports may not be needed when we implement a database
+/* TODO: Remove the imports below and static user data located at `./data` and
```

Comment:
Make a linear task where this TODO will be completed and change it to TODO(SWE-XX) where XX is the task id.
```suggestion
/* TODO(SWE-XX): Remove the imports below and static user data located at `./data` and
```